### PR TITLE
389 add torchvision resnet50 support

### LINKED ISSUE #389: Enable Torchvision resnet50 on PerceptualLoss
So far, `PerceptualLoss` does not support standard (torchvision based, https://pytorch.org/vision/main/models/generated/torchvision.models.resnet50.html) resnet50.
I will try to enable it. It would be helpful to use it as well as customized pretrained weights input.

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes #389 

This PR is used to enable torchvision's version of resnet50 on `PerceptualLoss`.
Users can also determine which pretrained weights is used.

### CODE REVIEW

File: generative/losses/perceptual.py [hunk 1]
```diff
@@ -65,8 +79,15 @@ def __init__(
             self.perceptual_function = MedicalNetPerceptualSimilarity(net=network_type, verbose=False)
```
Comment (yiheng-wang-nv) [inline generative/losses/perceptual.py:79]: My further thought is that maybe in the next version, consider the network as an arg directly rather than instantiate it within this class is a better way for users to define their customized network for feature extraction (then calculate the loss).

Comment (yiheng-wang-nv) [reply generative/losses/perceptual.py:79]: Hi @Warvito @ericspod , what do you think?

Comment (ericspod) [reply generative/losses/perceptual.py:79]: We should have a facility to allow users to provide their own networks yes. We could have an argument to the constructor `perceptual_function` which if not `None` will be used as the `self.perceptual_function` value, otherwise the existing choices will be made.

Comment (yiheng-wang-nv) [reply generative/losses/perceptual.py:79]: Thanks @ericspod !

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv @ericspod ,

May I know the status of this PR? Can we merge it now.

Thanks.

**Comment by yiheng-wang-nv** [comment]
> Hi @yiheng-wang-nv @ericspod ,
> 
> May I know the status of this PR? Can we merge it now.
> 
> Thanks.

It's done. I'm not sure is there any other requirements to merge a PR in this repo (it's my first contribution), may need @ericspod and @Warvito 's confirm, thanks!

**Comment by Nic-Ma** [comment]
Hi @ericspod ,

As this PR is necessary for the new MONAI 1.2 bundle and tutorial for Generative AI, I will merge it right now.
If you have any other comments, let's do it in a following PR.

Thanks.

**Comment by ericspod** [comment]
Hi @Nic-Ma, no problem. I had suggested an enhancement we can follow up with later if there's need.
